IPython/core/magics/code.py

```diff
@@ -258,25 +258,39 @@ def pastebin(self, parameter_s=''):
 
           -d: Pass a custom description. The default will say
               "Pasted from IPython".
+          -e: Pass number of days for the link to be expired.
+              The default will be 7 days.
         """
-        opts, args = self.parse_options(parameter_s, 'd:')
+        opts, args = self.parse_options(parameter_s, "d:e:")
 
         try:
             code = self.shell.find_user_code(args)
         except (ValueError, TypeError) as e:
             print(e.args[0])
             return
 
+        expiry_days = 7
+        if "e" in opts:
+            try:
+                expiry_days = int(opts.get("e", 7))
+            except ValueError as e:
+                print(e.args[0])
```

Comment:
```suggestion
                print(e.args[0].capitalize())
```

For consistency with L280.